locale/en_US.json

```diff
@@ -0,0 +1,62 @@
+{
+    "未命名对话历史记录": "Unnamed Dialog History",
+    "在这里输入": "Type in here",
+    "🧹 新的对话": "🧹 New Dialogue",
+    "🔄 重新生成": "🔄 Regeneration",
+    "🗑️ 删除最旧对话": "🗑️ Delete oldest dialog",
+    "🗑️ 删除最新对话": "🗑️ Delete latest dialog",
+    "模型": "Model",
+    "多账号模式已开启，无需输入key，可直接开始对话": "Multi-account mode is enabled, no need to enter key, you can start the dialogue directly",
+    "**发送消息** 或 **提交key** 以显示额度": "**Send message** or **Submit key** to display credit",
+    "选择模型": "Select Model",
+    "选择LoRA模型": "Select LoRA Model",
+    "实时传输回答": "Stream output",
+    "单轮对话": "Single-turn dialogue",
+    "使用在线搜索": "Use online search",
+    "选择回复语言（针对搜索&索引功能）": "Select reply language (for search & index)",
+    "上传索引文件": "Upload index file",
+    "双栏pdf": "Two-column pdf",
+    "识别公式": "OCR",
```

Comment:
should be "formula OCR" maybe?

Reply:
Indeed, you are right. My mistake.
"formula OCR" is correct.